packages/v/visualboyadvance-m/abi_symbols

```diff
@@ -1,8 +1,2 @@
-vbam:BC
-vbam:PC
-vbam:SP
 vbam:main
-visualboyadvance-m:BC
-visualboyadvance-m:PC
-visualboyadvance-m:SP
```

Comment:
Alarming. See above.

Reply:
Started up as expected. No errors in terminal.